mlos_bench/mlos_bench/tests/services/remote/ssh/conftest.py

```diff
@@ -6,118 +6,11 @@
-            compose_project_name=docker_compose_project_name,
-            service_name=SSH_TEST_SERVER_NAME,
-            hostname=ssh_test_server_hostname,
-            username='root',
-            id_rsa_path=id_rsa_file.name)
-        wait_docker_service_socket(docker_services, ssh_test_server_info.hostname, ssh_test_server_info.get_port())
-        id_rsa_src = f"/{ssh_test_server_info.username}/.ssh/id_rsa"
-        docker_cp_cmd = f"docker compose -p {docker_compose_project_name} cp {SSH_TEST_SERVER_NAME}:{id_rsa_src} {id_rsa_file.name}"
-        cmd = run(docker_cp_cmd.split(), check=True, cwd=os.path.dirname(__file__), capture_output=True, text=True)
-        if cmd.returncode != 0:
-            raise RuntimeError(f"Failed to copy ssh key from {SSH_TEST_SERVER_NAME} container: {str(cmd.stderr)}")
-        os.chmod(id_rsa_file.name, 0o600)
-        yield ssh_test_server_info
-        # NamedTempFile deleted on context exit
-
-
-@pytest.fixture(scope="session")
-def alt_test_server(ssh_test_server: SshTestServerInfo,
-                    docker_services: DockerServices) -> SshTestServerInfo:
-    """
-    Fixture for getting the second ssh test server info from the docker-compose.yml.
-    See additional notes in the ssh_test_server fixture above.
-    """
-    # Note: The alt-server uses the same image as the ssh-server container, so
-    # the id_rsa key and username should all match.
-    # Only the host port it is allocate is different.
-    alt_test_server_info = SshTestServerInfo(
-        compose_project_name=ssh_test_server.compose_project_name,
-        service_name=ALT_TEST_SERVER_NAME,
-        hostname=ssh_test_server.hostname,
-        username=ssh_test_server.username,
-        id_rsa_path=ssh_test_server.id_rsa_path)
-    wait_docker_service_socket(docker_services, alt_test_server_info.hostname, alt_test_server_info.get_port())
-    return alt_test_server_info
-
-
-@pytest.fixture
-def ssh_host_service(ssh_test_server: SshTestServerInfo) -> SshHostService:
-    """Generic SshHostService fixture."""
-    return SshHostService(
-        config={
-            "ssh_username": ssh_test_server.username,
-            "ssh_priv_key_path": ssh_test_server.id_rsa_path,
-        },
-        global_config={},
-        parent=None,
-    )
-
-
-@pytest.fixture
-def ssh_fileshare_service() -> SshFileShareService:
-    """Generic SshFileShareService fixture."""
-    return SshFileShareService(
-        config={
-            # Left blank to make sure we test per connection overrides.
-        },
-        global_config={},
-        parent=None,
-    )
+import mlos_bench.tests.services.remote.ssh.fixtures as ssh_fixtures
+
+# Expose those as local names.
+ssh_test_server_hostname = ssh_fixtures.ssh_test_server_hostname
+ssh_test_server = ssh_fixtures.ssh_test_server
+alt_test_server = ssh_fixtures.alt_test_server
+ssh_host_service = ssh_fixtures.ssh_host_service
+ssh_fileshare_service = ssh_fixtures.ssh_fileshare_service
```

Comment:
why not just
```suggestion
from mlos_bench.tests.services.remote.ssh.fixtures import (
    ssh_test_server_hostname, ssh_test_server, alt_test_server, ssh_host_service, ssh_fileshare_service
)
```

Reply:
This doesn't work unfortunately.  They fixtures need to be local variables to be picked up.